### PR TITLE
Newsletter 12

### LINKED ISSUE #12: Newsletter 12
Reporting issue for the next newsletter, roughly scheduled for `2018-09-23`.

### PR DESCRIPTION
**Pull request by jamesmunns**
Closes #12

### CODE REVIEW

File: content/2018-09-25-newsletter-12.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+draft = false
+in_search_index = true
+template = "page.html"
++++
+
+This is the twelfth newsletter of the [Embedded WG] where we highlight new progress, celebrate cool projects, thank the community, and advertise projects that need help!
+
+[Embedded WG]: https://github.com/rust-embedded/wg
+
+Discuss on [internals.rust-lang.org], [on twitter], or [on reddit]!
```
Comment (japaric) [inline content/2018-09-25-newsletter-12.md]: Perhaps we should post this to u.r-l.o under the embedded category?

File: content/2018-09-25-newsletter-12.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+[Driver Crates Released]: https://github.com/rust-embedded/awesome-embedded-rust#driver-crates
+[Driver Crates WIP]: https://github.com/rust-embedded/awesome-embedded-rust#wip
+[no-std crates]: https://github.com/rust-embedded/awesome-embedded-rust#no-std-crates
+
+## Help Wanted
+
+* Take a minute to check out the [wg help wanted issues] as labeled on our issue tracker!
+* Interested in implementing a new target for Embedded Rust? Nuvoton has a [development board] for the Cortex-M23. Check out the [thumbv8m tracking issue] if you want to help!
+
+[wg help wanted issues]: https://github.com/search?q=org%3Arust-embedded+is%3Aissue+is%3Aopen+milestone%3Arc1+label%3A%22help+wanted%22&type=Issues
```
Comment (japaric) [inline content/2018-09-25-newsletter-12.md]: This link returns an empty list. The milestone has been renamed from rc1 to edition, but there are no help-wanted edition issues either :-).

We could either remove this or link to *all* help-wanted issues within the org but that includes repos like cross, which I'm probably not to going to look at until next year.

File: content/2018-09-25-newsletter-12.md (hunk 1 above)
Comment (therealprof) [reply content/2018-09-25-newsletter-12.md]: +1

File: content/2018-09-25-newsletter-12.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+<!-- more -->
+
+If you want to mention something in [the next newsletter], make sure to leave a comment on the issue.
+
+[the next newsletter]: https://github.com/rust-embedded/blog/issues/15
+
+## Highlights
+
+* We now have a category for Embedded on the [users.rust-lang.org] forum! This is a great place to ask larger questions, or to share more details about your projects!
+* You can now build embedded [binaries on stable] with the `1.30-beta` release of Rust! Check it out and let us know if you run in to any issues with your projects!
```
Comment (therealprof) [inline content/2018-09-25-newsletter-12.md]: This could mention how to best let us know.

File: content/2018-09-25-newsletter-12.md (hunk 1 above)
Comment (jamesmunns) [reply content/2018-09-25-newsletter-12.md]: Will change

File: content/2018-09-25-newsletter-12.md (hunk 2 above)
Comment (jamesmunns) [reply content/2018-09-25-newsletter-12.md]: Will remove, :tada: :tada: :tada: !

File: content/2018-09-25-newsletter-12.md (hunk 3 above)
Comment (jamesmunns) [reply content/2018-09-25-newsletter-12.md]: Adding a link to the /wg issue tracker

**Comment by jamesmunns** [comment]
Argh, hit merge instead of bors.

**Comment by jamesmunns** [comment]
@japaric made changes to master, ok to re-enable branch protection.

**Comment by japaric** [comment]
branch protection has been re-enabled

**Pull request merged**